Drawer/Drawer.js

```diff
@@ -2,7 +2,7 @@
  * An Agate-themed drawer component.
  *
  * @example
- * <Drawer header="Surprise">Hello Enact!</Drawer>
+ * <Drawer style={{marginTop:'60px'}} open>Hello Enact!</Drawer>
```

Comment:
I think height it also needed to prevent cut off.
    margin-top: 33px;
    height: 100px;

Reply:
Need to use not px but ri.pixelToRem